app/main.py

```diff
@@ -1,8 +1,62 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int) -> None:
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+        self.washing_price = 0
+        self.income = 0
+        self.rate = 0
+
+    def calculate_washing_price(self, car: Car) -> float:
+        self.washing_price = (car.comfort_class
+                              * (self.clean_power - car.clean_mark)
+                              * self.average_rating
+                              / self.distance_from_city_center)
+        self.washing_price = round(self.washing_price, 1)
+
+        return self.washing_price
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        car_count = 0
+
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                washing_price = self.calculate_washing_price(car)
+                car_count += 1
+                self.income += washing_price
```

Comment:
```suggestion
                self.income += self.calculate_washing_price(car)
```
And you can delete the `washing_price` variable.
try to write less code. The less code, the better)